server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/SetVacationResponseMethod.java

```diff
@@ -101,7 +102,7 @@ public SetVacationResponseMethod(VacationRepository vacationRepository, Notifica
         if (vacationResponse.isValid()) {
             return vacationRepository.modifyVacation(accountId, vacationResponse.getPatch())
                 .then(notificationRegistry.flush(accountId))
-                .thenMany(Flux.just(JmapResponse.builder()
+                .thenMany(Mono.just(JmapResponse.builder()
```

Comment:
No `then`?

Reply:
?

Reply:
it does not do what I think it was doing, I'm fine with it